rules/Php71/Rector/FuncCall/RemoveExtraParametersRector.php

```diff
@@ -143,6 +152,34 @@ private function hasVariadicParameters(array $parameterAcceptors): bool
             }
         }
 
+        if ($functionReflection instanceof PhpFunctionReflection) {
+            $pathsFunctionName = explode('\\', $functionReflection->getName());
+            $functionName = array_pop($pathsFunctionName);
+
+            $fileName = (string) $functionReflection->getFileName();
+            /** @var Node[] $contentNodes */
+            $contentNodes = $this->parser->parse($this->smartFileSystem->readFile($fileName));
+
+            /** @var Function_ $function */
+            $function = $this->betterNodeFinder->findFirst($contentNodes, function (Node $node) use (
+                $functionName
+            ): bool {
+                if (! $node instanceof Function_) {
+                    return false;
+                }
+
+                return $this->isName($node, $functionName);
+            });
+
+            return (bool) $this->betterNodeFinder->findFirst($function->stmts, function (Node $node): bool {
+                if (! $node instanceof FuncCall) {
+                    return false;
+                }
+
+                return $this->isNames($node, ['func_get_args', 'func_num_args', 'func_get_arg']);
+            });
+        }
```

Comment:
:+1: 

Could you extract this to own service? We use similar variadic logic in multiple rules.

Reply:
implemented https://github.com/rectorphp/rector-src/pull/244/commits/04897b61123671024e11ed10a9bd83eedf157af9